modules/boards/config/routes.rb

```diff
@@ -3,14 +3,18 @@
 
   resources :boards,
             controller: 'boards/boards',
-            only: %i[new create],
+            only: %i[index show new create destroy],
             as: :work_package_boards
 
-  scope '', as: :work_package_boards do
-    get '/boards(/*state)', to: 'boards/boards#index'
-  end
-
   scope 'projects/:project_id', as: 'project' do
-    get '/boards(/*state)', to: 'boards/boards#index', as: :work_package_boards
+    resources :boards,
+              controller: 'boards/boards',
+              only: %i[index new show],
+              as: :work_package_boards
+
+    # Adding the `create` action into the above `resources` macro would
+    # result in a name collision between it and `get /boards(/*state)`
+    # as it would result in both being named `project_work_package_boards`
+    post '/boards', to: 'boards/boards#create'
```

Comment:
I'm not sure I follow, isn't the naming expected? You did remove the `boards/(*state)` route after all, so `show` now also has the same route helper?

Reply:
It's a stale comment. The issue would arise before I split `boards/(*state)` out into individual `index` and `show` actions. This "special case route" can even be removed an added in into the `resources` macro :)